test/core/handshake/client_ssl.cc

```diff
@@ -197,8 +197,9 @@ static void server_thread(void* arg) {
 
   // Wait until the client drops its connection.
   char buf;
-  while (SSL_read(ssl, &buf, sizeof(buf)) > 0)
+  while (SSL_read(ssl, &buf, sizeof(buf)) > 0) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done